### ISSUE TITLE
Peer Dependency Issue w/ Octokit

### ISSUE THREAD

**Issue by RichiCoder1** [ghiscoding/lerna-lite#258]
### Describe the bug

There appears to be a peer deps mismatch with one of `@lerna-lite/cli`'s dependencies:
![image](https://user-images.githubusercontent.com/2391878/179422429-b026cac2-a7a0-4c3c-a1f7-6565a57f021d.png)

I imagine a recent NPM w/ strict peer deps will likely run into a similiar issue.

### Expectation

Installs without warnings.

### Reproduction

Install `@lerna-lite/cli` into a fresh, pnpm 7.5.x package.

### Lerna config and logs

N/A

### Environment Info

```shell
Environment info:

  System:
    OS: Linux 5.10 Ubuntu 20.04.4 LTS (Focal Fossa)
    CPU: (12) x64 AMD Ryzen 5 5600G with Radeon Graphics
  Binaries:
    Node: 16.16.0 - ~/.asdf/installs/nodejs/16.16.0/bin/node
    npm: 8.11.0 - ~/.asdf/plugins/nodejs/shims/npm
  Utilities:
    Git: 2.25.1 - /usr/bin/git

PNPM via Corepack: `pnpm@7.5.2`
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/lerna-lite/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/ghiscoding/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
most probably already addressed, but not yet released, by Renovate PR #251

**Comment by ghiscoding** [comment]
this should be fixed in the latest release [v1.8.0](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.8.0) (note that I skipped `v1.7.0` by mistake)
